.github/workflows/continuous-delivery.yml

```diff
@@ -89,7 +89,7 @@ jobs:
           mvn ${{ env.MAVEN_CLI_OPTS }} versions:set -DnewVersion=${{ env.NEW_VERSION }}
 
       - name: Perform build with xvfb
-        uses: coactions/setup-xvfb@v1
+        uses: coactions/setup-xvfb@90473c3ebc69533a1a6e0505c36511b69c8c3135
```

Comment:
Ist das Absicht hier mit der Commit Nr.?

Reply:
ja weil es da kein vernünftiges Release gibt. wurde so auch in anderen Projekten gemacht.